setup.py

```diff
@@ -15,8 +16,10 @@ def get_contents(filename):
     return out
 
 #Make Home Directory
-if not os.path.exists(HOME_DIR):
-    os.makedirs(HOME_DIR)
+if os.path.exists(HOME_DIR):
+    shutil.rmtree(HOME_DIR)
```

Comment:
o_O
Are you aware of what this does? @DarkSouL11 

This removes all the directories in "~"